tests/logictest/suites/base/05_ddl/05_0003_ddl_alter_database

```diff
@@ -38,75 +38,43 @@ statement ok
 ALTER DATABASE IF EXISTS c RENAME TO a;
 
 statement query T
-SELECT name FROM system.databases;
```

Comment:
Can't guarantee previous tests file don't create databases and drop it.

So the test will be flaky..., from the context, I guess we only should test if database `a` and `b` are created.

So I changed it to `SELECT * FROM system.databases where name = 'a' or name = 'b';` which is more focused.

Reply:
Suggest to add where name ='information_schema'. I have change in this issue https://github.com/datafuselabs/databend/pull/8517

Reply:
The file is aim to test `alter` database. The test is to test the databases(a / b) that were created above. So I think we don't need to add `where name ='information_schema'`.

Reply:
Ok. I see same test in tests/logictest/suites/base/01_system/01_0003_information_schema